bitrise.yml

```diff
@@ -585,6 +597,50 @@ workflows:
                 exit 1
             fi
         is_always_run: true
+  Decision_to_run_UI_Tests:
+    steps:
+    - script@1:
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -e
+
+            echo "Commit on origin/main"
+            COMMIT_ON_MAIN_ORIGIN=`git log -n 1 origin/main --pretty=format:"%H"`
+            echo $COMMIT_ON_MAIN_ORIGIN
```

Comment:
Just double checking if this decision model always intended for `main` branch as you target here?

Reply:
So far, yes. Only checking against main, but once we have this running, we could add more branches